extensions/HuggingFaceTransformers/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2023 LastMile AI
```

Comment:
do we need to state MIT License? https://github.com/lastmile-ai/aiconfig/blob/main/LICENSE#L1